packages/loader/container-loader/src/containerContext.ts

```diff
@@ -181,12 +183,12 @@ export class ContainerContext implements IContainerContext {
         /** @returns clientSequenceNumber of last message in a batch */
         public readonly submitBatchFn: (batch: IBatchMessage[]) => number,
         public readonly submitSignalFn: (contents: any) => void,
+        public readonly disposeFn: (error?: ICriticalContainerError) => void,
```

Comment:
I'm surprised this didn't break the type tests.  Ok it looks like ContainerContext isn't included in the type tests for the package.  I guess that's because it's not exported.

Anyway, the "break" is fine - same as Container class - because the new one can be used in place of the old one, and most importantly, still adheres to the interface.